packages/fxa-payments-server/src/lib/errors.ts

```diff
@@ -1,9 +1,109 @@
 // ref: fxa-auth-server/lib/error.js
-export const AuthServerErrno = {
```

Comment:
I usually prefer leaving the exports in-line, rather than listing them all at the end. 

That said... there are a lot of intermediate variables here that *aren't* exported. Listing them at the end seems to keep them from getting lost in the rest of the file. So, :+1: